packages/nextlove/src/with-route-spec/middlewares/with-validation.ts

```diff
@@ -182,7 +183,7 @@ export const withValidation =
       (req.method === "POST" || req.method === "PATCH") &&
       (input.jsonBody || input.commonParams) &&
       !req.headers["content-type"]?.includes("application/json") &&
-      !isEmpty(req.body)
+      !_.isEmpty(req.body)
```

Comment:
I don't understand this file- why do you need to import all of lodash?

Reply:
seam-connect was complaining about not being able to find the isEmpty